Airports/EGKB/Stars.txt

```diff
@@ -36,20 +36,7 @@ STAR:EGKB:21:SUMUM1X:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV
 STAR:EGKB:03:SUMUM1X:SUMUM TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:21:XAMAN1X:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
 STAR:EGKB:03:XAMAN1X:XAMAN TEVMO GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-
-;Withdrawn 2019/12
-STAR:EGKB:21:#GODLU1A:BEDEK BIG UMTUM GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#GODLU1A:BEDEK BIG UMTUM GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:21:#GODLU1C:KONAN GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#GODLU1C:KONAN GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:21:#GODLU1D:SOVAT ERKEX OKVAP GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#GODLU1D:SOVAT ERKEX OKVAP GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:21:#GODLU1F:NEVIL OSPOL NETVU SOXUX OKVAP GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#GODLU1F:NEVIL OSPOL NETVU SOXUX OKVAP GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:21:#GODLU1J:GIBSO BEGTO AVANT BIG UMTUM GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
-STAR:EGKB:03:#GODLU1J:GIBSO BEGTO AVANT BIG UMTUM GODLU ELMIV LCE11 LCE12 LCE13 RAVSA GAPGI ATPEV LCE07 OSVEV
```

Comment:
Hi @AndyTWF what is the rationale for removing these?

Reply:
Committed for some reason that I don't know... I'll put them back!

Reply:
Ah yes, meant to fix one character problem (comment was a colon not semi) and I just deleted the whole lot... My bad :D